src/components/SinglePastCommitInfo.tsx

```diff
@@ -159,47 +166,50 @@ export class SinglePastCommitInfo extends React.Component<
       return <div>{this.props.trans.__('Error loading commit data')}</div>;
     }
     return (
-      <CommitDiff
-        actions={
-          <>
-            <ActionButton
-              className={actionButtonClass}
-              icon={discardIcon}
-              title={this.props.trans.__(
-                'Revert changes introduced by this commit'
-              )}
-              onClick={this._onRevertClick}
-            />
-            <ActionButton
-              className={actionButtonClass}
-              icon={rewindIcon}
-              title={this.props.trans.__(
-                'Discard changes introduced *after* this commit (hard reset)'
-              )}
-              onClick={this._onResetClick}
-            />
-            <LoggerContext.Consumer>
-              {logger => (
-                <ResetRevertDialog
-                  open={this.state.resetRevertDialog}
-                  action={this.state.resetRevertAction}
-                  model={this.props.model}
-                  logger={logger}
-                  commit={this.props.commit}
-                  onClose={this._onResetRevertDialogClose}
-                  trans={this.props.trans}
-                />
-              )}
-            </LoggerContext.Consumer>
-          </>
-        }
-        numFiles={this.state.numFiles}
-        insertions={this.state.insertions}
-        deletions={this.state.deletions}
-        files={this.state.modifiedFiles}
-        onOpenDiff={this.props.onOpenDiff}
-        trans={this.props.trans}
-      ></CommitDiff>
+      <div>
+        <p className={commitBodyClass}>{this.state.commitBody}</p>
```

Comment:
I find it nice to display the detailed only when expanding the detailed. This keeps the first view as compact as possible.

For the format, you probably need to replace the `\n` with `<br />`. Probably something like:

```ts
const body = [];
this.state.commitBody.split('\n').forEach(line => { body.push(line, <br />) });

// ...

<p className={commitBodyClass}>{body}</p>
```